_docs/06-best-practice-pull-request.md

```diff
@@ -73,11 +73,15 @@ Push your branch to Github:
 6. When you are satisfied and want to push it to Apache's remote repo proceed with Merging a PR
 
 # How to create a PR (contributors)
-Create pull requests: [GitHub PR docs](https://help.github.com/articles/creating-a-pull-request/).
+Before you create a pull request, make sure
+1. A corresponding [JIRA](https://issues.apache.org/jira/browse/ROCKETMQ/) issue is created and has a clear problem description.
+2. You follow [Coding Guidelines](http://rocketmq.apache.org/docs/code-guidelines/).
```

Comment:
Maybe use a relative path is better, like:
`You follow \[Coding Guidelines\](/docs/code-guidelines/).`